system-tests/projects/react16/cypress/support/component.js

```diff
@@ -0,0 +1,4 @@
+import { mount, unmount } from 'cypress/react16'
```

Comment:
I can't find a reference to this system test project `react16`. Also, we aren't publishing `react16`.

Reply:
Oops, yep.

Reply:
Gone